packages/voicechat-server/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "voicechat-server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "ts-node-dev ./src/index.ts",
+    "build": "./node_modules/typescript/bin/tsc"
```

Comment:
Because not every user will have typescript installed globally. But every user will have typescript inside ```node_modules``` because it's a dependency.

Reply:
In the scripts field, you can refer to tsc without a global install.